src/commonMain/kotlin/data/itemsets/Deathmantle.kt

```diff
@@ -23,14 +26,12 @@ class Deathmantle : ItemSet() {
     fun noCostConsumeProc(buff: Buff): Proc {
         return object : Proc() {
             override val triggers: List<Trigger> = listOf(
-                Trigger.ROGUE_CAST_KIDNEY_SHOT,
-                Trigger.ROGUE_CAST_EVISCERATE,
-                Trigger.ROGUE_CAST_ENVENOM,
-                Trigger.ROGUE_CAST_RUPTURE,
-                Trigger.ROGUE_CAST_SLICE_AND_DICE,
+                Trigger.ROGUE_CAST_FINISHER
             )
             override val type: Type = Type.STATIC
 
+            // TODO: is this also consumed on misses/dodges/parry etc? otherwise we have to check here for a hit
+            // if it does consume on a miss/etc. we need to check interactions with the "Quick Recovery" talent.
```

Comment:
See comment. If it does consume on misses, "Quick Recovery" will refund energy when it shouldn't with the current implementation.